service/prometheus/prometheus.go

```diff
@@ -1,27 +1,15 @@
 package prometheus
 
 import (
-	"strings"
-
 	"k8s.io/client-go/pkg/api/v1"
 )
 
 const (
-	CertificateAnnotation = "giantswarm.io/prometheus-config-controller/certificate"
-	ClusterAnnotation     = "giantswarm.io/prometheus-config-controller/cluster"
+	ClusterAnnotation = "giantswarm.io/prometheus-config-controller/cluster"
 )
 
-// GetCertificateName returns the value of the certificate annotation, splitted by '/'.
-// Assumed that the service contains this annotation, see `FilterInvalidServices`.
-func GetCertificateName(service v1.Service) (string, string) {
-	reference := service.ObjectMeta.Annotations[CertificateAnnotation]
-	parts := strings.Split(reference, "/")
-
-	return parts[0], parts[1]
-}
-
-// GetGroupName returns the value of the cluster annotation.
+// GetClusterID returns the value of the cluster annotation.
 // Assumed that the service contains this annotation, see `FilterInvalidServices`.
-func GetGroupName(service v1.Service) string {
+func GetClusterID(service v1.Service) string {
 	return service.ObjectMeta.Annotations[ClusterAnnotation]
```

Comment:
This code looks like it can just panic accidentally. Lets prevent this.

Reply:
This also looks like belonging to `key` package <3

Reply:
+1 but Joe had a different opinion on this.

Reply:
Yeah, but Joe's not certain he's right now :D

Reply:
My initial thinking was it makes sense in `prometheus` as it's very specific to the annotation, but not certain anymore. Might change later?